### DISCUSSION TITLE
Install keeps saying me there is no INVENTREE_DB_USER

### DISCUSSION THREAD

**Discussion by Shinusagi** [inventree/InvenTree#8044] [Q&A] [answered]
Hello people, I am really, REALLY sorry to bother you but I am losing already 3 days now with the instalation, after so many issues I decided to go with Docker which already made me lose more time but at least it is working... however...

When I follow the install placed on the same directory where I moved the 3 Files from the repo to install with Docker and try this on the console:

docker compose run --rm inventree-server invoke update

I get the following message you can see on the attachment 
![GitHelpTree](https://github.com/user-attachments/assets/eb68ea15-bb2b-4f08-9a39-35e19961cbc3)

Honestly I am so lost, I tried from a different directory and I get a different message because the files are not there, so it is recognizing the .dev file but not reading as it should on the variables there? I am lost and burnt out at this point, please I really want to use InvenTree but I need I can't even past that installation error and I thought Docker was going to be the easier way... I just started learning Docker because of InvenTree >_> and I was able to fully install Docker on Mint but still can't install InvenTree, please help... and thanks.

**Comment by matmair** [comment, marked as answer]
Your environment file must be named `.env` in a default setup - that is the standard for docker and set [here](https://github.com/inventree/InvenTree/blob/cc45c239155834855db602dc88e9f5d6c123ec49/contrib/container/docker-compose.yml#L74-L75) in the docke-compose.yaml.
You can read more about that [here](https://docs.docker.com/compose/environment-variables/set-environment-variables/#use-the-env_file-attribute) if you are interested in the mechanism in docker-compose.

**Comment by Shinusagi** [comment]
OMG it was so stupid... the file for some reason downloaded as env.txt so I deleted the ".txt" and was named only "env", then still having issues someone mentioned the file with ".env extension" so made me thing I should put a name.env" on it
Now it seems to be working, thanks a lot, A LOT

**Reply by matmair** [reply]
Many browsers do not support downloading a file without a filename and rename it automatically. The .env name is often used for "secret" files as there are passwords etc in there and those should not be shared easily so a lot of software is cautious when handling them. This is a somewhat common pitfall with docker / Linux server stuff, I would not sweat over it

**Reply by breadx333** [reply]
this helped me
